FiegeIntegration/Integration/01_ManageFiegeConnection.md

```diff
@@ -0,0 +1,120 @@
+
+5. Click the [SAVE] button.  
+    The connection will be established. The *Checking credentials...* notice is displayed. After a few seconds, the *Loading data...* notice is displayed.
+
+    ![Checking credentials](../../Assets/Screenshots/Fulfillment/Settings/Connections/CheckingCredentials.png "[Checking credentials]")
+
+    The *Create connection* view is automatically closed when the connection has been established. The *Edit connection* view of the newly created connection is displayed. The *Settings* tab is selected.
+
+    ![Connection](../../Assets/Screenshots/Fulfillment/Settings/Connections/FiegeIntegration/EditConnectionSettings_WarehouseIDFallback.png "[Connection]")
+
+    > [Info] By default, all connections that have been established are first inactive and therefore must be enabled after the connection configuration.  
+
+6. Continue to configure the connection, see [Configure the Fiege connection](#configure-the-fiege-connection).
+
+
+## Configure the Fiege connection
+
+After the Fiege connection has been established, further settings can be configured.
+
+#### Prerequisites
+
+- A Fiege connection has been established, see [Create a Fiege connection](#create-a-fiege-connection).
+- The warehouses to be managed by Fiege have been configured in the *Warehouse* module. 
+
+#### Procedure
+
+*Fulfillment > Settings > Tab CONNECTIONS*
+
+![Fiege connection](../../Assets/Screenshots/Fulfillment/Settings/Connections/FiegeIntegration/Connections.png "[Fiege connection]")
+
+1. Click the Fiege connection to be configured in the list of connections.  
+The *Edit connection* view is displayed. By default, the *Credentials* tab is selected.
+
+    ![Edit connection credentials](../../Assets/Screenshots/Fulfillment/Settings/Connections/FiegeIntegration/EditConnectionCredentials.png "[Edit connection credentials]")
+
+2. Click the *Settings* tab.   
+    The *Settings* tab is displayed. By default, the *Warehouse ID fallback* menu entry is selected.
+
+    ![Settings warehouse ID fallback](../../Assets/Screenshots/Fulfillment/Settings/Connections/FiegeIntegration/EditConnectionSettings_WarehouseIDFallback.png "[Settings warehouse ID fallback]")
+
+3. Enter the number of the warehouse to be used as fallback warehouse for stock movements if no other reference is found. 
+
+    > [Info] The warehouse numbers configured in the Core1 can be found in the *Warehouse* module: *Warehouse > Settings > Warehouse master data > Select a warehouse > Basic data*
+
+4. Click the *Bearer token* menu entry in the left side bar.  
+    The bearer token settings are displayed on the right side.  
+
+    ![Settings bearer token](../../Assets/Screenshots/Fulfillment/Settings/Connections/FiegeIntegration/EditConnectionSettings_BearerToken.png "[Settings bearer token]")
+
+5. Enter the bearer token for authentication with the Fiege API as provided by Fiege in the *Bearer token* field.
```

Comment:
```suggestion
5. Enter the bearer token for authentication with the Fiege API in the *Bearer token* field.
```
Würde ich weglassen - schon in der Einleitung geschrieben und der Satz wird hier sonst zu umständlich
Alternativ als Info danach einfügen